src/main/resources/lang/dutch.lang

```diff
@@ -0,0 +1,190 @@
+		warning details: <yellow>    %s<reset>\n
+		error details: <light red>    %s<reset>\n
+		other details: <white>    %s<reset>\n
+		line details: <gold>    Lijn: <gray>%s<reset>\n <reset>
+
+		config, aliases and scripts: de configuratie, aliassen en alle scripts
+		scripts: alle scripts
+		main config: de hoofdconfiguratie
+		aliases: de aliassen
+		script: <gold>%s<reset>
+		scripts in folder: alle scripts in <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <lime>script¦¦s¦ in <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> bevat geen ingeschakelde scripts.
+	enable:
+		all:
+			enabling: Alle uitgeschakelde scripts inschakelen...
+			enabled: Alle eerder uitgeschakelde scripts ingeschakeld en geparseerd
+			error: <gold>%s <light red>error¦¦s¦ <light red>tegengekomen bij het parseren van uitgeschakelde scripts!
+			io error: <light red>Kon een of meer scripts niet laden - sommige scripts zijn mogelijk al hernoemd en zullen worden ingeschakeld wanneer de server opnieuw opstart: <gold>%s
+		single:
+			already enabled: <gold>%s<reset> is al ingeschakeld! gebruik <gray>/<gold>skript <cyan>reload <red>%s<reset> om het te herladen als het aangepast is.
+			enabling: <gold>%s<reset> inschakelen...
+			enabled: <gold>%s<reset> succesvol ingeschakeld en geparseerd.
+			error: <gold>%2$s <light red>error¦¦s¦ <light red>tegengekomen bij het parseren van <gold>%1$s<light red>!
+			io error: <light red>Kon <gold>%s <light red>niet inschakelen: <gold>%s
+		folder:
+			empty: <gold>%s<reset> bevat geen uitgeschakelde scripts.
+			enabling: <gold>%2$s<reset> script¦¦s¦ in <gold>%1$s<reset> inschakelen...
+			enabled: <gold>%2$s<reset> eerder uitgeschakelde scripts in <gold>%1$s<reset> succesvol ingeschakeld en geparseerd.
+			error: <gold>%2$s <light red>error¦¦s¦ <light red>tegengekomen bij het parseren van scripts in <gold>%1$s<light red>!
+			io error: <light red>Fout bij het inschakelen van een of meer scripts in <gold>%s <light red>(sommige scripts worden mogelijk ingeschakeld wanneer de server opnieuw opstart): <gold>%s
+	disable:
+		all:
+			disabled: Alle scripts met succes uitgeschakeld.
+			io error: <light red>Kon een of meer scripts niet hernoemen - sommige scripts zijn mogelijk al hernoemd en worden uitgeschakeld wanneer de server opnieuw wordt opgestart: <gold>%s
+		single:
+			already disabled: <gold>%s<reset> is al uitgeschakeld!
+			disabled: <gold>%s<reset> succesvol uitgeschakeld.
+			io error: <light red>Kon <gold>%s<light red> niet hernoemen, het wordt weer ingeschakeld wanneer je de server opnieuw opstart: <gold>%s
+		folder:
+			empty: <gold>%s<reset> bevat geen ingeschakelde scripts.
+			disabled: <gold>%2$s<reset> script(s) in <gold>%1$s<reset> succesvol uitgeschakeld.
+			io error: <light red>Kon een of meerdere scripts in <gold>%s <light red>niet uitschakelen <light red>(sommige scripts worden mogelijk uitgeschakeld wanneer de server opnieuw opstart): <gold>%s
```

Comment:
not sure you should use `Kon [...]`, imo too similar to a direct English translation

Reply:
While I see your point, I disagree. `Kon` is the best, imo, translation for `Could` without changing the meaning of what is being said.

Reply:
that's fair, I didnt add a suggestion as I wasn't sure on how to improve it, but i think changing the sentence structure to be more like something like this would help in making it sound less like a direct translation.

```suggestion
			io error: <light red>Een of meerdere scripts in <gold>%s konden <light red>niet worden uitgeschakeld <light red>(sommige scripts worden mogelijk uitgeschakeld wanneer de server opnieuw opstart): <gold>%s
```

Reply:
Sure, that keeps the same meaning while not being a direct translation, I'll commit it.